### PR TITLE
Support setting attribute values for dropdown & multiselect.

### PR DESCRIPTION
**Pull request by piotrgrundas**
[Saleor PR](https://github.com/mirumee/saleor/pull/7037)

**PR intended to be tested with API branch:** 

### Screenshots
image
image

### Pull Request Checklist

1. [x] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [x] Attributes `[data-test-id]` are added for new elements
4. [x] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Review by dominik-zeglen — APPROVED** [review]

File: src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx [hunk 1]
```diff
@@ -11,19 +11,31 @@ import ConfirmButton, {
...
 import { buttonMessages } from "@saleor/intl";
-import { maybe } from "@saleor/misc";
 import { getFormErrors } from "@saleor/utils/errors";
+import omit from "lodash/omit";
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
+import slugify from "slugify";
 
 import { AttributeDetails_attribute_values } from "../../types/AttributeDetails";
+import * as M from "./messages";
```
Comment (mmarkusik) [inline src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx:25]: Why called `M` ? we call it `messages` everywhere else, let's stick to that

File: src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx [hunk 2]
```diff
@@ -45,47 +57,66 @@ const AttributeValueEditDialog: React.FC<AttributeValueEditDialogProps> = ({
...
+                    error={!!formErrors[field]}
+                    fullWidth
+                    helperText={getAttributeValueErrorMessage(
+                      formErrors[field],
+                      intl
+                    )}
+                    name={field as keyof AttributeValueEditDialogFormData}
+                    label={intl.formatMessage(message)}
+                    value={data[field]}
+                    onChange={evt => {
```
Comment (mmarkusik) [inline src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx:109]: We call it `event` everywhere else, let's stick to that. Also, please move onChange outside of return

File: src/attributes/components/AttributeValues/AttributeValues.tsx [hunk 3]
```diff
@@ -90,16 +90,22 @@ const AttributeValues: React.FC<AttributeValuesProps> = ({
...
             <TableCell className={classes.columnStore}>
               <FormattedMessage
-                defaultMessage="Default Store View"
+                defaultMessage="Storefront Name"
                 description="attribute values list: name column header"
               />
             </TableCell>
+            <TableCell className={classes.columnStore}>
+              <FormattedMessage
+                defaultMessage="Value"
```
Comment (mmarkusik) [inline src/attributes/components/AttributeValues/AttributeValues.tsx:105]: Export to messages

**Review by eCzerniak — CHANGES_REQUESTED** [review]
1. At Create Attribute page when user fill value field and save all, then values are gone
2. At Product/Page view - it should be shown `Admin Name` not `Storefront Name`.(screenshot) 
image

**Pull request closed without merging**